src/overlays/actors/ovl_En_Bal/z_en_bal.c

```diff
@@ -4,6 +4,20 @@
  * Description: Tingle with Balloon
  */
 
+/*
+Week Event Flags:
+        10 Bit 0 -- Whether player has talked to Tingle before during this cycle
+        35 Bits 0-5	--	Whether player has bought each possible map. For a given map, the flag is 1 << mapId.
+        64 Bits 3-4 --  Appears to be a 2 bit enum that stores player form when 10/0 is set. 0 - Zora, 1 - Deku, 2 -
+Goron, 3 - Human Notably neither bit is ever cleared by EnBal, so this field could be corrupted if not cleared at the
+same time as 10/0
+
+Message IDs:
+    0x1D00 - 0x1D17
+    1D0E - 1D0F don't appear at all (explicitly)
+    1D10 is checked for, but never set or used explicitly
```

Comment:
I'm not sure how useful is to have the message ids of the actor listed here.
Also, checking on the MM text spreadsheet seems like `0x1D0E`, `0x1D0F` and `0x1D10` don't exist at all, which is kinda weird